cmd/sync.go

```diff
@@ -201,12 +201,11 @@ var syncCmd = &cobra.Command{
 				}
 				continue
 			}
-			if err == nil {
-				//wg.Wait()
-			} else {
+			if err != nil {
 				PrintError(err.Error())
 			}
 		}
+		wg.Wait()
```

Comment:
does it work for `Delete` and `LocalDelete`? is it locked? should we add `wg.Add(1)` there?

Reply:
@cnlangzi wg is added with statusBar only. we have added this were the process is in saperate go-routine and we not waiting for the process to be completed. were as in delete and localDelete it is synchronous only. at some points we are calling some internal methods parallelly but main methods (delete and localDelete) still remain synchronous only. so in that method calls we don't need to add the wg in that method.